src/globus_cli/parsing/shared_options/id_args.py

```diff
@@ -0,0 +1,39 @@
+"""
+These are semi-standard arg names with overridable metavars.
+
+Basic usage:
+
+>>> @endpoint_id_arg
+>>> def command_func(endpoint_id: uuid.UUID):
+>>>     ...
```

Comment:
Is this pulled out by anything, like autodoc? It may be worthwhile to call this out as:

```rst
..  code-block:: pycon

    >>> @endpoint_id_arg
    ... def command_func(endpoint_id: uuid.UUID):
    ...     ...
```

If not, carry on! 😀

Reply:
Yeah, it's not autodoc'ed. I was sort of tempted to remove it when I moved these docs around, but I figure it's not doing any harm and it might help someone get a quick read on these helpers. 😄